src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -7105,6 +7107,356 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  treatment_anatomic_sites:
+    $ref:
+      - "_terms.yaml#/treatment_anatomic_site/common"
```

Comment:
should the ref be plural too? I mean "_terms.yaml#/treatment_anatomic_site**s**/common"

Reply:
I looked into this earlier.  Usually for plurals, the ref is identical to the singular, minus the plural title, so I decided to leave this one singular.  

We could add the plural ref to _terms.yaml, but I vote we add it after we merge the other changes, since there are so many moving parts here, all of which point at _terms.yaml.

Reply:
sgtm!